packages/docs-infra/docs/next.config.mjs

```diff
@@ -0,0 +1,35 @@
+import createMDX from '@next/mdx';
+
+/** @type {import('next').NextConfig} */
+const nextConfig = {
+  pageExtensions: ['js', 'jsx', 'md', 'mdx', 'ts', 'tsx'],
+  output: 'export',
+  turbopack: {
+    rules: {
+      './app/**/demos/*/index.ts': {
+        as: '*.ts',
+        loaders: ['@mui/internal-docs-infra/loadPrecomputedCodeHighlighter'],
```

Comment:
Does this work with webpack as well, assuming `next build` is building using webpack ?

Reply:
Yes, it runs with webpack during `next build` and turbopack during `next dev`

Because `next build` with turbopack is not stable yet